itou/users/models.py

```diff
@@ -431,7 +431,7 @@ def latest_approval(self):
 
         if not approvals:
             return None
-        if approvals.valid().exists():
+        if approvals.valid():
             return approvals.valid().first()
```

Comment:
```suggestion
        valid_approval = approvals.valid().first()
        if valid_approval is not None:
            return valid_approval
```
ne permettrait pas d'en faire encore moins ?

Reply:
Effectivement, `first()` ajoute un `LIMIT 1` à la requête. Autrement, on chargeait tous les PASS et renvoyait juste le premier.